### ISSUE TITLE
compute job_id only after extracting passed file into the appropriate parameter

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/dispatcher-app#185]
the issue is that first job_id is computed with this parameter 'use_scws': 'user_file' while after it's converted to scw_list:[.... So it's because of the special case of file passing. Our test should include this case, @burnout87 . I make a separate issue.

This issue here diverted off-track - the roles of @ferrigno are correctly implemented once correct token is used.

**Comment by volodymyrss** [comment]
@burnout87  does this make sense?

**Comment by burnout87** [comment]
Yes it totally makes sense, and I see the problem.

**Comment by burnout87** [comment]
Found the problem:

- In the first request (the one initiated with the click the Submit button), the file is attached to the `POST` request, and `use_scws: user_file`
- In the second request (the one that fails and issues the 403 error), `use_scws` is still `user_file`, but the list file is not attached, rather the parameter `scw_list` is set 

This causes the issue of the different `job_id`, hence the 403 error

**Comment by burnout87** [comment]
So, the `job_id` is calculated the very first time without the `scw_list` parameter, but then this parameter is added and stored in the `analysis_parameters.json`.

Afterwards, the `job_ib` will be calculated with the `scw_list` (since the analysis_parameters.json file is used), but the `job_id` provided will be the one calculated without the `scw_list`, hence the issue.

**Comment by burnout87** [comment]
I have a fix in mind, I will make a PR and we'll if it's a good approach

**Comment by burnout87** [comment]
Fixed with #190 and #187
